OneSignalSDK/onesignal/src/main/java/com/onesignal/ActivityLifecycleHandler.java

```diff
@@ -50,7 +50,10 @@
         void available(@NonNull Activity activity) {
         }
 
-        void stopped(WeakReference<Activity> reference) {
+        void stopped() {
+        }
+
+        void lostFocus() {
         }
```

Comment:
Do all of these functions have no body? Is this okay?

Reply:
yeah it's for the inheritance classes to implemente